meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "duck" %}
```

Comment:
Draft conda recipe, the code from this branch can be installed with `conda install openduck -c gaintherapeutics`. It would be nice to submit the recipe to conda-forge at some point.